ship.config.js

```diff
@@ -2,6 +2,13 @@ const fs = require("fs");
 const path = require("path");
 
 module.exports = {
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
```

Comment:
If it's a patch update and there is no fix (which means only `chore` commits), then we do not prepare a release pull-request. This prevents unnecessary pull-requests every week.

Reply:
So if I understand correctly, this will prevent Renovate from triggering new release PRs?

Reply:
Even before `shouldPrepare` is called, Ship.js checks if the latest commit message starts with "chore: release v", and thus renovate commits won't trigger new releases.